File: Metody_Implementacji_Algorytmów/Lista12/zad1.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int n;
    cin >> n;
    for(int i=0; i<n; i++){
        int a, b, c;
        cin >> a >> b >> c;
        int second = c/2 + b;
        int first = c - c/2 + a;
        if(first > second)
            cout << "First" << endl;
        else
            cout << "Second" << endl;
    }
}

File: Programowanie_Pod_Windows(.NET)/Lista4/Zad1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Zad1
{
    public static class StringExtensions
    {
        public static bool IsPalindrome(this string s)
        {
            string s2 = new string(s.Where(c => !char.IsPunctuation(c) && !char.IsWhiteSpace(c)).ToArray()).ToLower();
            
            string first = s2.Substring(0, s2.Length / 2);
            char[] arr = s2.ToCharArray();
            Array.Reverse(arr);
            string temp = new string(arr);
            string second = temp.Substring(0, temp.Length / 2);
            return first.Equals(second);
        }
    }
    internal class Program
    {
        static void Main(string[] args)
        {
            string s = "Kobyła ma mały bok.";
            bool ispalindrome = s.IsPalindrome();
            Console.WriteLine(ispalindrome);
            Console.ReadLine();
        }
    }
}
